### ISSUE TITLE
Change the header text color from white to black (#000000) for accessibility

### ISSUE THREAD

**Issue by katz95** [spotify/backstage#306]
Ran a color contrast test and the white text doesn't pass - changing it to #000000 will help with accessibility. 

<img width="799" alt="Screenshot 2020-03-18 at 3 21 47 PM" src="https://user-images.githubusercontent.com/61153904/76970474-395b8c80-692c-11ea-977e-d2d15a3b8640.png">

**Comment by Rugvip** [comment]
What about darkening the background? :grin:

**Comment by stefanalund** [comment]
Yeah...I'm not loving the black text either :)

**Comment by katz95** [comment]
Accessible versions - let me know which one you prefer! Also explored it in the blade pattern

<img width="970" alt="Screenshot 2020-03-19 at 11 24 11 AM" src="https://user-images.githubusercontent.com/61153904/77057415-34521800-69d4-11ea-8402-95ae49244588.png">

**Comment by freben** [comment]
Overlaps with #261

**Comment by Rugvip** [comment]
@katz95 <span>#</span>2! what are the color codes? :>

**Comment by katz95** [comment]
@Rugvip just saw this!! 😅
![Wave Deconstruction](https://user-images.githubusercontent.com/61153904/77542299-42a5a580-6ea6-11ea-89ff-3fc63df94bfb.png)

**Comment by mateusmarquezini** [comment]
Guys, assign this issue to me, please? Thx! :tada:

**Comment by Rugvip** [comment]
@katz95 would it be possible to get the color codes of the gradient without any opacity? Ideally the input should just be two colors, one light and one dark.

**Comment by katz95** [comment]
hmm @Rugvip  there's some layering that happens with opacity, but let me try! Also, if just sent an SVG of the complete header itself (not separate elements), would that work?
